src/components/scene.js

```diff
@@ -205,12 +239,16 @@ export class Scene extends Component {
                 useDevicePixels={useDevicePixels}
                 style={style}
                 views={new MapView({repeat: true})}
+                // views={views}
                 /* onBeforeRender={this._onBeforeRender} // Not yet
                       onHover={visStateActions.onLayerHover} // Not yet
                       onClick={visStateActions.onLayerClick}*/ // Not yet
-                {...this.props.adapter.getProps(this.deckgl, () => {}, () => {this.forceUpdate()})}
+                {...this.props.adapter.getProps(this.deckgl, () => {}, () => {this.forceUpdate()})} /* @Chris My understanding (from Elisa) is that this is forced because this.deckgl is not defined initially */
+                // {...this.props.adapter._updateFromProps()}
               >
-                 
+                 <View id="map">
```

Comment:
I’m not sure what you’re trying to achieve with the View here or the views object above. While we’ll eventually use this to render the basemap from mapbox we shouldn’t need it yet for the timestamp - that all seems to be handled with deck.gl layers.

Reply:
👍